posthog/hogql_queries/insights/trends/breakdown.py

```diff
@@ -0,0 +1,118 @@
+from typing import List, Tuple
+from posthog.hogql import ast
+from posthog.hogql_queries.insights.trends.breakdown_values import BreakdownValues
+from posthog.hogql_queries.insights.trends.utils import series_event_name
+from posthog.hogql_queries.utils.query_date_range import QueryDateRange
+from posthog.models.filters.mixins.utils import cached_property
+from posthog.models.team.team import Team
+from posthog.schema import ActionsNode, EventsNode, TrendsQuery
+
+
+class Breakdown:
```

Comment:
Would be good to add return types to all the functions here, and makes our lives a bit easier later.